hyeongjin/pintos/userprog/select_test.sh

```
@@ -0,0 +1,381 @@
+      tokens+=("$n")
+    done
+  else
+    tokens+=("$tok")
+  fi
+done
+
+declare -A seen=()
+sel_tests=()
+for n in "${tokens[@]}"; do
+  if [[ "$n" =~ ^[0-9]+$ ]] && (( n>=1 && n<=${#tests[@]} )); then
+    idx=$((n-1))
```

Shellcheck (0.10.0)

[warning] idx appears unused. Verify use (or export if used externally).

(SC2034)